kythe/go/extractors/cmd/gotool/gotool.go

```diff
@@ -70,6 +84,7 @@ Options:
 	flag.StringVar(&bc.GOROOT, "goroot", bc.GOROOT, "Go system root")
 	flag.BoolVar(&bc.CgoEnabled, "gocgo", bc.CgoEnabled, "Whether to allow cgo")
 	flag.StringVar(&bc.Compiler, "gocompiler", bc.Compiler, "Which Go compiler to use")
+	flag.Var(&arrayFlags{array: &bc.BuildTags}, "buildtag", "Go +build tags to enable during extraction. May be specified multiple times.")
 
```

Comment:
"buildtags"?

Also, what about plumbing them through the build?

Reply:
Renamed to "buildtags".

What do you mean by "plumbing them through the build"? The extractor does already handle these correctly, it just didn't have a way to specify them on the command line.

Reply:
This extractor doesn't actually execute a build—it only simulates one. When running `go build` explicitly one includes tags in the same way, via a flag.

When building in Bazel, tags are handled differently: Bazel permits mixing packages in a single directory, so the the rules have to do some shenanigans to virtualize the directory structure and I believe file selection is done there.